upgrade/breaking-changes/7-0-0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 7.0.0
+description: Handle the changes in the 7.0.0 release version of the Telerik UI for .NET MAUI components.
+page_title: Breaking Changes in 5.0.0
+slug: changes-in-7-0-0
+position: 1
+---
+
+# Breaking Changes in 7.0.0
+
+This article lists the breaking changes included in Telerik UI for .NET MAUI 7.0.0 release version.
+
+## Common Changes
+
+This section applies to changes to multiple components.
```

Comment:
The changes described in the Common Changes section apply to multiple components.